netplay/createroom.cpp

```diff
@@ -109,7 +115,6 @@ void CreateRoom::onFinished(int)
     (*CoreDoCommand)(M64CMD_ROM_CLOSE, 0, NULL);
     if (!launched && webSocket)
     {
-        disconnect(webSocket, SIGNAL(error(QAbstractSocket::SocketError)), this, SLOT(handleConnectionError(QAbstractSocket::SocketError)));
```

Comment:
the `error` signal for web sockets is a bit too finnicky to rely on - it is fired when the socket is properly closed which can result in problems. So I've switched over to just using the QTimers to verify that the connection happened or not.